framework/src/media/streaming/player.c

```diff
@@ -24,8 +24,8 @@
 #include <assert.h>
 #include <pthread.h>
 #include <debug.h>
-#include <audiocodec/streaming/player.h>
-#include "internal_defs.h"
+#include "player.h"
+#include "../utils/internal_defs.h"
```

Comment:
please check this. we can remove '../'

Reply:
I tried, building failed without `../` :
CC:  src/media/streaming/player.c
src/media/streaming/player.c:28:33: fatal error: utils/internal_defs.h: No such file or directory
 #include "utils/internal_defs.h"